src/app/portlets/content-types/fields/content-type-fields-row-list/content-type-fields-row-list.component.ts

```diff
@@ -1,7 +1,7 @@
 import { Component } from '@angular/core';
 import { FieldService, FieldDragDropService } from '../service';
 import { DragulaService } from 'ng2-dragula';
```

Comment:
This it's not being used here and also in content-type-fields-row-list.component and content-types-layout.component